src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,16 @@
+.footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffe9e0;
+  color: #230903;
+  height: 50px;
+  width: 100vw;
+  padding: 5px 10px;
+  text-align: left;
+  font-family: sans-serif, Calibri;
+  font-size: 14px;
```

Comment:
So if you want to start with the accessibility we could do it. But baby steps. Let's start with using REMs for font-size. Nowhere else, do I want you to get the feeling for pixels, because they are much more common. People don't know what accessibility even means.

Some reading on this topic - https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/